### ISSUE TITLE
Alt- Text for Graphs

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/cas#254]
**[Original report](https://bitbucket.org/laborstudents/cas-flask/issue/254) by Sarah Watts (Bitbucket: [sarahawatts](https://bitbucket.org/sarahawatts), GitHub: [SarahAWatts1](https://github.com/SarahAWatts1)).**

----------------------------------------

Alt text needs to be created for the home screen on CAS. If the description of the graph is larger than 250 characters than the alt attribute in HTML cannot be used. Because the graph falls under a complex image you will most likely have to provide alt text somewhere else other than the alt attribute. 

Below is a link that will help (The graph falls under "Complex Images" so look for that header):

https://webaim.org/techniques/alttext/

**Comment by bryantal** [comment]
**Original comment by Jamal Aw Yonis (Bitbucket: [awyoonisj](https://bitbucket.org/awyoonisj), GitHub: [jawyoonis](https://github.com/jawyoonis)).**

----------------------------------------

@sheggen do we need to adjust the graphs in CAS still?

**Comment by bryantal** [comment]
**Original comment by Scott Heggen (Bitbucket: [sheggen](https://bitbucket.org/sheggen), GitHub: [sheggen](https://github.com/sheggen)).**

----------------------------------------

Yes. Or, you need to check them and determine otherwise.
